pro/src/pages/CollectiveOffer/CollectiveOffer/components/OfferEducational/OfferEducationalForm/OfferEducationalForm.tsx

```diff
@@ -147,38 +156,39 @@ export const OfferEducationalForm = ({
             <FormOfferType
               domainsOptions={domainsOptions}
               nationalPrograms={nationalPrograms}
-              disableForm={mode === Mode.READ_ONLY}
+              disableForm={canEditDetails}
```

Comment:
Je m'attendrais à ça plutôt, non ?
```suggestion
              disableForm={!canEditDetails}
```